server/graphql/asset/src/types.rs

```diff
@@ -241,13 +267,111 @@ impl From<AssetLogFilterInput> for AssetLogFilter {
         AssetLogFilter {
             id: f.id.map(EqualFilter::from),
             asset_id: f.asset_id.map(EqualFilter::from),
-            status: f.status.map(StringFilter::from),
+            status: f.status.map(|s| map_filter!(s, StatusInput::to_domain)),
             log_datetime: f.log_datetime.map(DatetimeFilter::from),
             user: f.user.map(StringFilter::from),
         }
     }
 }
 
+#[derive(Enum, Copy, Clone, PartialEq, Eq, Debug, Serialize)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")] // only needed to be comparable in tests
+
+pub enum ReasonInput {
```

Comment:
Is this too generic as a name for graphql type ? What if we have other reasons, or statuses for that matter